### DISCUSSION TITLE
articles/the-new-poor

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#177] [Comments]
# articles/the-new-poor

《新约·马太福音》中说：“富人进入天堂比骆驼穿过针眼还要难。”古往今来，无论是民间传说，还是文学作品，富人大多是以贪婪、吝啬、狡诈等负面形象出现，而穷人大多正直、善良，嫉恶如仇。也就是说，道德高尚者往往以穷人的身份出现，他们安贫乐道，视金钱如粪土。从某种意义上说，贫穷是一件美德。这符合广大老百姓的期望，因为在漫长的几千年的人类文明史中，大部分人都是穷人。

https://hutusi.com/articles/the-new-poor

**Comment by NOeSIS-yu** [comment]
杰克马，哈哈哈~

**Comment by wangyu-xreal** [comment]
写的很不错

**Comment by hutusi** [comment]
三年前读了三十年前鲍曼写的《工作、消费主义和新穷人》，写下此文。现在看来，依然不过时。写下此文后不久，就发生了两件大事。一是国家全面放开疫情管控，因为疫情的不确定性变成“穷人”的不在少数。二是ChatGPT的发布，当时只惊叹于其强大的生成能力；而近来越来越多人担心工作会被AI取代，进而变成消费主义社会的“无用之人”，也就是书中说的“新穷人”。

今天，正好是ChatGPT发布三周年。
